packages/react/ds-core/src/ui/Button/Button.stories.tsx

```diff
@@ -76,7 +76,7 @@ export const Custom: Story = {
  */
 export const IntentsInheritance: Story = {
   decorators: [
-    (Story) => (
+    (_Story) => (
```

Comment:
Is this because the storybook API expects a function that accepts an argument, but Biome throws an error if you have unused parameters unless they are prefixed by _?